app/views/common/_page_title.html.erb

```diff
@@ -1,4 +1,7 @@
-<%= render "govuk_publishing_components/components/title", {
```

Comment:
If there are no other title components in this app you can remove the Sass include as well: https://github.com/alphagov/search-admin/blob/main/app/assets/stylesheets/application.scss#L23

Reply:
Good spot thanks, now done.